code/modules/surgery/organ_manipulation.dm

```diff
@@ -15,7 +15,7 @@
 		)
 
 /datum/surgery/organ_manipulation/soft
-	possible_locs = list(BODY_ZONE_PRECISE_GROIN, BODY_ZONE_PRECISE_EYES, BODY_ZONE_PRECISE_MOUTH, BODY_ZONE_L_ARM, BODY_ZONE_R_ARM)
```

Comment:
> People were using it for unintended situations like suiciding as clerk and taking the hud

I feel removing a feature due to players abusing part of it isn't the best way to approach the problem.

Reply:
This is hardly a feature. Was only used to abuse Clerk HUDs, and nothing else. If it becomes necessary to add again we will.

Reply:
Here's a few responses from community members on the matter.
![image](https://user-images.githubusercontent.com/77302679/204898274-0efcea04-1f89-4047-af48-7b0fb711eef7.png)
![image](https://user-images.githubusercontent.com/77302679/204898385-ed34215d-a468-4a4f-9186-3549f087bdee.png)
![image](https://user-images.githubusercontent.com/77302679/204898412-2d4e1318-2adc-40d8-8bde-b0b0398342e9.png)

We just simply have no need for this feature as it only is used for unintended exploits. This is a hotfix PR meant to soft remove two really annoying features.